spec/features/vote_comment_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe 'Voting on a Comment', type: :feature do
+  context "with an article with existing comemnts" do
+    let!(:user) {create(:user)}
+    let!(:article) {create(:article, user: user)}
+    let!(:comment) {create(:comment, commentable: article, user: user)}
+
+    before do
+      visit article_path(article)
+    end
+
+    it 'does not let you vote without signed in user' do
+      page.find("#vote-comment-up-nil-#{comment.id}").click
+      within('.comment-vote') do
+        expect(find('.comment-vote-count')).to have_content(0);
+      end
+    end
+
+    context 'with a signed in user' do
+      let!(:vote_user) {login_user(name: 'voteuser', email: 'voteuser@viget.com')}
```

Comment:
always use fake emails in test, just in case.
`*@example.com` is reserved by [RFC 2606](https://tools.ietf.org/html/rfc2606) so you can use that in your tests.